packages/spor-react/src/datepicker/DatePicker.tsx

```diff
@@ -84,13 +92,19 @@ export const DatePicker = forwardRef<HTMLDivElement, DatePickerProps>(
       ref as React.MutableRefObject<HTMLDivElement>
     );
 
-    const styles = useMultiStyleConfig("Datepicker", {});
+    const styles = useMultiStyleConfig("Datepicker", {variant});
     const locale = useCurrentLocale();
 
+    //deprecated
     const responsiveVariant =
       useBreakpointValue(typeof variant === "string" ? [variant] : variant) ??
       "simple";
-    const hasTrigger = responsiveVariant === "with-trigger";
+      const hasTrigger = responsiveVariant === "with-trigger"
+      //midlertidid løsning for å ikke lage breakingchange
+      const hasNewTrigger = responsiveVariant === "base" ||
+      responsiveVariant === "floating" ||
+      responsiveVariant === "ghost";
+    
```

Comment:
I think we can remove the `simple` design now, without making it a breaking change. So you can delete the `responsiveVariant` code, and if the `variant` is `simple`, coerce it into being `base`.

And now all datepickers has a trigger, so this code in particular can be deleted.